docs/source/cpp/build_system.rst

```diff
@@ -111,33 +111,32 @@ Other available packages
-+-----------------------------------------------+--------------------------------------------------------+-----------------------------------------------------+
-| ``find_package(ArrowFlightSql REQUIRED)``     | ``ArrowFlightSql::arrow_flight_sql_shared``            | ``ArrowFlightSql::arrow_flight_sql_static``         |
-+-----------------------------------------------+--------------------------------------------------------+-----------------------------------------------------+
-| ``find_package(ArrowFlightTesting REQUIRED)`` | ``ArrowFlightTesting::arrow_flight_testing_shared``    | ``ArrowFlightTesting::arrow_flight_testing_static`` |
-+-----------------------------------------------+--------------------------------------------------------+-----------------------------------------------------+
-| ``find_package(ArrowSubstrait REQUIRED)``     | ``ArrowSubstrait::arrow_substrait_shared``             | ``ArrowSubstrait::arrow_substrait_static``          |
-+-----------------------------------------------+--------------------------------------------------------+-----------------------------------------------------+
-| ``find_package(ArrowTesting REQUIRED)``       | ``ArrowTesting::arrow_testing_shared``                 | ``ArrowTesting::arrow_testing_static``              |
-+-----------------------------------------------+--------------------------------------------------------+-----------------------------------------------------+
-| ``find_package(Gandiva REQUIRED)``            | ``Gandiva::gandiva_shared``                            | ``Gandiva::gandiva_static``                         |
-+-----------------------------------------------+--------------------------------------------------------+-----------------------------------------------------+
-| ``find_package(Parquet REQUIRED)``            | ``Parquet::parquet_shared``                            | ``Parquet::parquet_static``                         |
-+-----------------------------------------------+--------------------------------------------------------+-----------------------------------------------------+
+This is the list of available packages:
+
+* ArrowCUDA
+* ArrowAcero
+* ArrowCompute
+* ArrowDataset
+* ArrowFlight
+* ArrowFlightSql
+* ArrowFlightTesting
+* ArrowSubstrait
+* ArrowTesting
+* Gandiva
+* Parquet
+
+Target specific to above packages follow a consistent naming pattern:
+
+* find_package usage: ``find_package(PackageName REQUIRED)``
+* shared target: ``PackageName::package_name_shared``
+* static target: ``PackageName::package_name_static``
+
+Example, to use the ArrowFlightTesting package:
+
+* Package Name: ``ArrowFlightTesting``
+* find_package Usage: ``find_package(ArrowFlightTesting REQUIRED)``
+* Shared Target: ``ArrowFlightTesting::arrow_flight_testing_shared``
+* Static Target: ``ArrowFlightTesting::arrow_flight_testing_static``
```

Comment:
```suggestion
For example, to use the ``ArrowCompute`` package:

* find_package Usage: ``find_package(ArrowCompute REQUIRED)``
* Shared Target: ``ArrowCompute::arrow_compute_shared``
* Static Target: ``ArrowCompute::arrow_compute_static``
```

Reply:
An example is great, but I think a more common example like ArrowCompute would be even better.